src/index.ts

```diff
@@ -16,42 +16,44 @@ export async function submitForm(
 ) {
     // get the second element and last element from docPath split by "/"
     const userId = docPath.split("/")[1];
+    const formRef = push(ref(db, `forms/${userId}`));
+    await set(
+        ref(db, docPath),
```

Comment:
write to formRef not on the docPath path since it's the business logic and distributor that will right the document to docPath